src/main/java/ch/njol/skript/hooks/economy/classes/Money.java

```diff
@@ -42,14 +42,15 @@
 		Classes.registerClass(new ClassInfo<>(Money.class, "money")
 				.user("money")
 				.name("Money")
-				.description("A certain amount of money. Please note that this differs from <a href='#number'>numbers</a> as it includes a currency symbol or name, but usually the two are interchangeable, e.g. you can both <code>add 100$ to the player's balance</code> and <code>add 100 to the player's balance</code>.")
-				.usage("<code>&lt;number&gt; $</code> or <code>$ &lt;number&gt;</code>, where '$' is your server's currency, e.g. '10 rupees' or '£5.00'")
+				.description("A certain amount of money. Please note that this differs from <a href='#number'>numbers</a> as it includes a currency symbol or name, but usually the two are interchangeable, e.g. you can both add 100$ to the player's balance and add 100 to the player's balance.")
+				.usage("&lt;number&gt; $ or $ &lt;number&gt;, where '$' is your server's currency, e.g. '10 rupees' or '£5.00'")
```

Comment:
They served no purpose since they were added, in the original docs they didn't even work, and with the restyle they still serve no purpose with no visual change
![image](https://user-images.githubusercontent.com/20037329/155845896-fd8cd0d8-21f4-40fe-bec3-fde6dbe78fdd.png)
![image](https://user-images.githubusercontent.com/20037329/155845899-1f9fc7a9-c508-401d-a9be-4059e7b168e1.png)
![image](https://user-images.githubusercontent.com/20037329/155845907-1053cce4-3ec7-4f2d-ae50-169278018172.png)

Reply:
Description should keep that, not patterns :ok_hand: